RPIPs/RPIP-42.md

```diff
@@ -25,6 +25,14 @@ This proposal also explicitly tries to benefit the smallest NOs in a few ways, i
 - We give precedence to small nodes staking some number of initial validators in the Node Operator queue (see [RPIP-59](RPIP-59))
 - There is a small but tangible financial benefit for large stakers that stake as few large nodes instead of many small nodes -- this (alongside our vote power, which scales with the square root of vote-eligible RPL) helps preserve the strong governance voice of small NOs
 
+## Motivation
+
+The primary motivation for the introduction of smaller ETH bonds in this form is twofold. First, lower ETH bonds allow for the same amount of bonded ETH to support a larger volume of rETH, this helps unlock growth for the protocol, as it has been primarily limited by lack of Node Operators in the months preceding this proposal. Second, lower ETH bonds increase the profitability of Rocket Pool as a protocol, allowing it to grow and compete favorably with alternative liquid staking providers.
+
+The innovation of a bond curve as opposed to a flat smaller allowable ETH bond is motivated by the need to maintain a base-level stake to mitigate MEV theft, slashing penalties, and abandonment; while meeting the core goal of improved capital efficiency.
```

Comment:
```suggestion
The innovation of a bond curve is motivated by the need to maintain enough stake to mitigate MEV theft, slashing penalties, and abandonment; while meeting the core goal of improved capital efficiency.
```

Reply:
If the reader already knows what a bond curve is, then the "as opposed to" portion is unnecessary. If a reader doesn't know what a bond curve is, I don't think they will understand the concept just from "flat smaller allowable"...

If we wanted to provide a higher level explanation of what a bond curve is we could do that, but the explainer material already covers that, and then a more detailed description is found in the rest of the RPIP. So I think we can just omit as suggested

Reply:
Sure, makes sense. It was a bit of an awkward phrasing before as well.